### DISCUSSION TITLE
Require Sepolia ETH exhausted todays limit

### DISCUSSION THREAD

**Discussion by SuryaSundarVadali** [Cyfrin/foundry-full-course-cu#2190] [General]
can someone please donate me 0.2 Sepolia ETH 
address : 0x8271EE4B23577FF8ef4E831c183687DAb4b3D35c

**Comment by EngrPips** [comment]
I trust someone will help as soon as they can.

**Reply by SuryaSundarVadali** [reply]
I have a doubt i have deployed my nft on sepolia testnet as a jpg file the nft was minted but the image on the nft was not shown I read that only jpeg and png are supported so should I deploy again after changing the file format.

**Reply by EngrPips** [reply]
I will say yes.

**Reply by SuryaSundarVadali** [reply]
Ok thanks

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
